### PR TITLE
Release 2.2.0

### PR DESCRIPTION
**Pull request by GretaD**
Signed-off-by: greta <gretadoci@gmail.com>

### CODE REVIEW

**Comment by ChristophWurst** [comment]
Do you branch off into stable2.2 right after the merge or shall we do that before?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,47 @@
...
+- Improve the size of padding bottom
+- Provisioning config not saving
+- Sending PGP messages as HTML
+- Copy to clipboard
+- Envelope and message previews showing PGP ciphertext
+- Double scrollbar in thread view
+- Alias provisioning: Skip alias when identical to account email
+- Improve the dropdown multiselect menu of default folders
+- XOAUTH2 auth via SMTP
+- Disable provisioned accounts if using passwordless authentication
```
Comment (st3iny) [inline CHANGELOG.md:34]:
```suggestion
- Disable provisioned accounts if using passwordless authentication
- Fix dragover indicator being inherited to children
```

https://github.com/nextcloud/mail/pull/7728

Comment (st3iny) [reply CHANGELOG.md:34]: Depending on the answer to https://github.com/nextcloud/mail/pull/7727#issuecomment-1337252504.

**Comment by GretaD** [comment]
> Do you branch off into stable2.2 right after the merge or shall we do that before?

i was planning after, but since you're asking then let do that before

**Review by ChristophWurst — APPROVED** [review]

**Pull request merged**